session.go

```diff
@@ -219,7 +219,8 @@ func (s *session) setup(
 		return nil, nil, err
 	}
 
-	s.packer = newPacketPacker(s.connectionID, s.cryptoSetup, s.connectionParameters, s.streamFramer, s.perspective, s.version)
+	s.packer = newPacketPacker(s.connectionID, s.cryptoSetup, s.connectionParameters, s.streamFramer,
+		s.perspective, s.version)
```

Comment:
I'd like to keep to about 100 chars per line, so a single line is out. I find this more readable than putting each parameter on its own line (as long as it fits into two lines with this format). If you don't feel strongly about it, I'd prefer to keep it this way.

FWIW, I see this format often used in other languages (C++ / Java).